src/Core/Grid/Column/Type/Common/PositionColumn.php

```diff
@@ -72,6 +73,8 @@ protected function configureOptions(OptionsResolver $resolver)
             ->setAllowedTypes('update_method', 'string')
             ->setAllowedTypes('record_route_params', ['array'])
             ->setAllowedTypes('clickable', 'bool')
+            /* In some cases position needs to be incremented by one to make sense to merchant */
+            ->setAllowedTypes('increment_position', 'bool')
```

Comment:
I don't think we should make this behaviour configurable The input position in the grid should always follow the same convention to be 0-indexed or it will become a mess to handle specific use cases for each entity

If for now products were 1-indexed I think it's acceptable to change this and turn them into a 0-indexed position so that we uniformize this behaviour in all the BO

Reply:
If we are worried of changing the initial index for product position we can always decrement the grid data and adapt it in the controller or handler for example, but it should be transparent compared with the way the grid behaves

Besides I'm not sure but since the new command is based on the `PositionUpdater` service it might be 0-indexed by default anyway